app/assess/templates/assessor_tasklist.html

```diff
@@ -16,8 +16,10 @@
 
 <div class="govuk-phase-banner flex-parent-element">
     <p class="govuk-!-text-align-left flex-parent-element flexed-element-margins-collapse">
-        {{ govukBackLink({'href': url_for("assess_bp.landing"), 'text': 'Back to assessment dashboard'}) }}
+        <a href="{{url_for('assess_bp.landing')}}" class="govuk-back-link" id="back-to-assessment-dashboard">
+            Back to <b>assessment dashboard</b></a>
```

Comment:
Any reason from switching off the partial? not against it cause it gives us more flexibility but could've done:
```suggestion
        {{ govukBackLink({'href': url_for("assess_bp.landing"), 'html': 'Back to <b>assessment dashboard</b>', 'attributes': {'id': 'back-to-assessment-dashboard'}}) }}
```

Reply:
We also have this on 4 pages, so I'd either:

- make sure these changes are applied to all locations
- refactor out into a partial and change all locations them to use that

The two locations not on this PR but on main are:
https://github.com/communitiesuk/funding-service-design-assessment/blob/c548325b1ad08d2640d955be21c3be91b5eb5362/app/assess/templates/continue_assessment.html#L11

https://github.com/communitiesuk/funding-service-design-assessment/blob/c548325b1ad08d2640d955be21c3be91b5eb5362/app/assess/templates/resolve_flag.html#L11